docs/reference/setup/install/connect-clients.asciidoc

```diff
@@ -5,9 +5,11 @@ HTTP layer. A CA certificate is generated and stored on disk, and the hex-coded
 SHA-256 fingerprint of this certificate is also output to the terminal. Any 
 clients that connect to {es}, such as the 
 https://www.elastic.co/guide/en/elasticsearch/client/index.html[{es} Clients],
-{beats}, {ls}, and {fleet} must validate that they trust the certificate that
-{es} uses for HTTPS. Clients can establish trust by using either the fingerprint
-of the CA certificate or the CA certificate itself.
+{beats}, standalone {agent}s, and {ls} must validate that they trust the
+certificate that {es} uses for HTTPS. {fleet-server} and {fleet}-managed
+{agent}s are automatically configured to trust the fingerprint of the CA.
```

Comment:
```suggestion
{agent}s are automatically configured to trust the CA certificate.
```

The CA certificate is what is ultimately trusted. Whether the actual CA certificate  or the fingerprint of said CA certificate is used to do that, is more of an internal implementation detail for us. I think that expressing this as "trusting the certificate" matches terminology that users are used to read and is the least probable to raise any eyebrows.